--- a/browser/shell_views.cc	2016-03-15 12:37:21.786843719 +0000
+++ b/browser/shell_views.cc	2016-03-15 12:37:42.513142697 +0000
@@ -11,7 +11,7 @@
 #include "content/public/browser/web_contents.h"
 #include "content/public/common/context_menu_params.h"
 #include "content/shell/browser/shell_platform_data_aura.h"
-#include "device/bluetooth/bluetooth_adapter_factory.h"
+//#include "device/bluetooth/bluetooth_adapter_factory.h"
 #include "ui/aura/client/screen_position_client.h"
 #include "ui/aura/env.h"
 #include "ui/aura/window.h"
